documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1365,9 +1365,10 @@ The following configuration properties are _required_ unless a default value is
 |
 |Raw database jdbc url. This property can be used when more flexibility is needed and can support raw TNS names or RAC connection strings.
 
-|[[oracle-property-database-tablename-case-insensitive]]<<oracle-property-database-tablename-case-insensitive, `+database.tablename.case.insensitive+`>>
+|[[oracle-property-database-tablename-case-insensitive]]<<oracle-property-database-tablename-case-insensitive, `+database.tablename.case.insensitive+`>> +
```

Comment:
Let's remove it from the docs right away, so to avoid further proliferation of its usage. I'll do that and merge.